.github/workflows/test.yaml

```
@@ -45,5 +46,7 @@ jobs:
         run: cargo fmt -- --check
       - name: build-enclone
         run: cargo build
```

```suggestion
        run: cargo build --release
```
`cargo build` and `cargo test` need to both use `--release` or neither.